### ISSUE TITLE
Reproduce the pretraining of whisper style (OWSM-V1)

### ISSUE THREAD

**Issue by oggyfaker** [espnet/espnet#5943]
Hi , i really want to reproduce the pretraining of OWSM-V1 training with whisper style by your repo. I know this repo is very popular so a lot of question can be raised in the future, so i want to ask some questions as much as possible at once time  

I follow the instruction [link](https://github.com/espnet/espnet/tree/master/egs2/TEMPLATE/s2t1#1-data-preparation)
, but i can't find the way to train from scratch the model. Everything just about Fine-Tuning model.  

1. I want to ask about the instruction for training from scratch of OWSM-V1. Can you give me the clear instruction to do it ? 

2. The `run.sh` is very confuse for me , and this is the fisrt time i use espnet library , so can you show me the path of backbone and the loop training (backward and forward) model in `espnet/egs2/owsm_v1/s2t1` use for training ? 

3. The loss you use for OWSM-V1 is the combination of CTC loss and Cross-entropy loss(like training autoregressive model eg: GPT2) when training model from scratch ? 

4. I see the dataset for training is  the `Transcription aligned in utterance level` like this 

`<en><asr><0.00> Several years ago here at TED, Peter Skill- man introduced a design challenge called the marshmallow challenge.<5.60><5.80> And the idea’s pretty simple: Teams of four have to build the tallest free-standing structure out of 20 sticks of spaghetti, one yard of tape, one yard of string and a marshmallow.<16.52>`

So have you ever tried `Transcription aligned in word level` . If you tried can you share me the result you have ? 

`<en><asr><0.00> Several <0.02> <0.02> years <0.04> <0.04>ago<0.06> ...<eos>`

Thanks in advance !

**Comment by oggyfaker** [comment]
@pyf98 hope to see your answer ^^

**Comment by pyf98** [comment]
Hi @oggyfaker , thanks for reaching out!

1. OWSM training is similar to the other ASR or ST recipes in ESPnet. If you are new to ESPnet, I would suggest first trying one of the standard ASR recipes to understand the entire procedure of using ESPnet, e.g., https://github.com/espnet/espnet/tree/master/egs2/librispeech_100/asr1
There are also some tutorials available, e.g., https://espnet.github.io/espnet/notebook/ESPnet2/Course/CMU_SpeechRecognition_Fall2022/recipe_tutorial.html

2. The `s2t` model is here: https://github.com/espnet/espnet/blob/master/espnet2/s2t/espnet_model.py
The trainer is here: https://github.com/espnet/espnet/blob/master/espnet2/train/trainer.py
The `abs_task` and `s2t_task` are also relevant: https://github.com/espnet/espnet/blob/master/espnet2/tasks/abs_task.py
https://github.com/espnet/espnet/blob/master/espnet2/tasks/s2t.py

3. I found that using CTC and cross-entropy stabilized training (similar to normal ASR), so it's better to keep it.
4. We haven't officially used word-level timestamps yet, but it should also work quite well based on others' experience.
